metacat-connector-polaris/src/functionalTest/resources/schema.sql

```diff
@@ -12,7 +12,8 @@ create table TBLS (
   id uuid default gen_random_uuid() not null primary key,
   db_name varchar(255) not null,
   tbl_name varchar(255) not null,
-  metadata_location varchar(1024),
+  previous_metadata_location varchar(8192),
```

Comment:
Is `8192` the more correct upper bound size? If so we have to change it in the test crdb as well.

Reply:
Ah yeah, it's just a better guess. Thanks, updated it in test crdb as well.